app/src/main/java/com/practicum/playlistmaker/library/data/db/convertors/TrackDbConvertor.kt

```diff
@@ -0,0 +1,41 @@
+package com.practicum.playlistmaker.library.data.db.convertors
+
+import com.practicum.playlistmaker.library.data.db.entity.TrackEntity
+import com.practicum.playlistmaker.search.domain.models.Track
+
+class TrackDbConvertor {
+
+    fun mapModelToEntity(track: Track): TrackEntity {
+        return TrackEntity(
+            track.trackName,
+            track.artistName,
+            track.trackTimeMillis,
+            track.artworkUrl100,
+            track.trackId,
+            track.collectionName,
+            track.releaseDate,
+            track.primaryGenreName,
+            track.country,
+            track.previewUrl,
+            track.id,
+            true
+        )
+    }
```

Comment:
```suggestion
    fun mapModelToEntity(track: Track): TrackEntity = TrackEntity(
            track.trackName,
            track.artistName,
            track.trackTimeMillis,
            track.artworkUrl100,
            track.trackId,
            track.collectionName,
            track.releaseDate,
            track.primaryGenreName,
            track.country,
            track.previewUrl,
            track.id,
            true
        )
```

Так можно писать для упрощения и читабельности